responsibleai/test-requirements.txt

```diff
@@ -0,0 +1 @@
+xgboost
```

Comment:
where is this test-requirement.txt used?
Aren't all pipeline based on requirement-dev

Reply:
Removed